app/src/main/java/com/yacgroup/yacguide/TourbookActivity.kt

```diff
@@ -345,8 +341,7 @@ class TourbookActivity : BaseNavigationActivity() {
     }
 
     private fun _selectExportFile(mimeType: String) {
-        val mimeTypeMap = MimeTypeMap.getSingleton()
-        val extension = mimeTypeMap.getExtensionFromMimeType(mimeType)
+        val extension = com.yacgroup.yacguide.utils.MimeTypeMap.getExtensionFromMimeType(mimeType)
```

Comment:
This method is not necessary if you pass a parameter of type `TourbookExportFormat` (instead of just the mine type) and consider my comment for extending the enum values by extension and mime type.